lib/libutils/isoc/strtok.c

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2022 Foundries Ltd <jorge@foundries.io>
+ */
+#include <stdlib.h>
+#include <string.h>
+
+char* strtok(char *str, const char *delims)
```

Comment:
Why not copy this function (or `strtok_r()` as suggested by @jforissier) from newlib instead?